pom.xml

```diff
@@ -322,7 +322,9 @@
 
             <properties>
                 <quarkus.package.type>native</quarkus.package.type>
-                <quarkus.native.builder-image>quay.io/quarkus/ubi-quarkus-mandrel:20.1.0.1.Final-java11</quarkus.native.builder-image>
+                <!-- TODO: Can't use `ubi-quarkus-mandrel` image due to https://github.com/graalvm/mandrel/issues/218 -->
+                <!-- <quarkus.native.builder-image>quay.io/quarkus/ubi-quarkus-mandrel:20.1.0.1.Final-java11</quarkus.native.builder-image> -->
```

Comment:
and the image issue is probably resolved, like one day ago or so :)

Reply:
@mjurc said to use the mandrel version from the pom.xml. Either I should remove this comment or use the mandrel version? What do you think?

Reply:
I was reacting to Mandrel issue which is resolved now as new images were published 18 hours ago.

"mandrel version from pom" ... you probably mean the default builder image defined in upstream Quarkus

I think we should continue with Mandrel here as it is the default for product.

@mjurc introduced Mandrel image in his PR https://github.com/quarkus-qe/quarkus-openshift-test-suite/pull/63, would be interested to hear what changed his mind wrt the default builder image :)

Reply:
I've changed this to use the mandrel 21.0 image version. Let me know if you want to use the version from the pom.xml instead.